start.sh

```diff
@@ -1,3 +1,12 @@
-#!/bin/sh
-export PYTHONPATH="$(dirname "$(readlink -f "${BASH_SOURCE[0]}" )" )"
-python meshroom/ui
+#!/bin/bash
+export MESHROOM_ROOT="$(dirname "$(readlink -f "${BASH_SOURCE[0]}" )" )"
+export PYTHONPATH=$MESHROOM_ROOT:$PYTHONPATH
+
+# using existing alicevision release
+#export LD_LIBRARY_PATH=/foo/Meshroom-2021.1.0/aliceVision/lib/
+#export PATH=$PATH:/foo/Meshroom-2021.1.0/aliceVision/bin/
+
+# using alicevision built source
+#export PATH=$PATH:/foo/build/Linux-x86_64/
+
+python3 "$MESHROOM_ROOT/meshroom/ui"
```

Comment:
Keep using the python defined by default in the environment.
```suggestion
python "$MESHROOM_ROOT/meshroom/ui"
```

Reply:
@fabiencastan On Ubuntu default command is `python3`. `python` command is not set. Which distro are you using? (otherwise it is needed to install `sudo apt install python-is-python3`)